.github/workflows/npmpublish.yml

```diff
@@ -32,6 +32,3 @@ jobs:
           node-version: 12
           registry-url: https://registry.npmjs.org/
       - run: yarn --frozen-lockfile
-      - run: yarn publish
```

Comment:
Can you exclude this file?

Reply:
Oops, yep.  Pushing to my fork complains on that (and CI fails) because I'm  not you.  I'll put it back for your tree.